driver/src/main/java/org/neo4j/driver/Logging.java

```diff
@@ -118,13 +122,27 @@ static Logging slf4j() {
         return new Slf4jLogging();
     }
 
+    /**
+     * Create logging implementation that uses {@link java.lang.System.Logger}.
+     *
+     * @return new logging implementation.
+     */
+    static Logging javaPlatformLogging() {
+        return new JavaPlatformLogging();
+    }
+
     /**
      * Create logging implementation that uses {@link java.util.logging}.
      *
      * @param level the log level.
      * @return new logging implementation.
+     * @throws IllegalStateException if SLF4J is not available.
```

Comment:
Shouldn't `SLF4J` be replaced with something like `java.util.logging`?

Reply:
Ah, I missed that. Fixed!